spec/views/item_seasons/show.html.erb_spec.rb

```diff
@@ -3,8 +3,8 @@
 require 'rails_helper'
 
 RSpec.describe 'item_seasons/show', type: :view do
-  let!(:item_season_produces) { ProduceItem.create(name: 'morango', category: 'fruit') }
-  let!(:item_season) { ItemSeason.create(produce_item_id: item_season_produces.id, month_index: 1, country_code: 'BR') }
+  let!(:item_season_produces) { create(:produce_item, name: 'morango') }
+  let!(:item_season) { create(:item_season, produce_item: item_season_produces) }
 
```

Comment:
@tcione I also changed this one to use factories, but I left line 13 as it was, as I'm not too sure about it. WDYT?

Reply:
Hm, you mean `ProduceItem.joins(:item_seasons).where(item_seasons: { month_index: item_season.month_index })`?

Reply:
yes 🙂

Reply:
Ah, Factories are just for creation/building. Fetching from database, update and delete should be done by normal Rails stuff.